include/rocksdb/options.h

```diff
@@ -348,6 +348,8 @@ struct DbPath {
   DbPath(const std::string& p, uint64_t t) : path(p), target_size(t) {}
 };
 
+static const std::string kDbHostId = "__hostname__";
```

Comment:
This name doesn't leave any room for alternatives. How about `kHostnameForDbHostId`?